### PR TITLE
add paddle.subtract, optimize paddle.maximum and paddle.minimum

### PR DESCRIPTION
**Pull request by LutaoChu**
### PR types

New features,Function optimization
### PR changes

APIs
### Describe

1 Add paddle.subtract API
2 Remove the argument 'axis' of paddle.maximum, paddle.minimum
3 Polish the docs of paddle.maximum, paddle.minimum
4 Fix the return value shape specification for ElementWise Ops

image
image
image
image
image
image

### CODE REVIEW

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -295,6 +296,67 @@ def add(x, y, name=None):
...
+
+    Returns:
+        N-D Tensor. A location into which the result is stored. Its dimension equals with $x$.
+
+    Examples:
+
+        .. code-block:: python
+        
+            import paddle
+            import numpy as np
```
Comment (wawltor) [inline python/paddle/tensor/math.py:312]: import numpy as np 放在前面

File: python/paddle/tensor/math.py [hunk 2]
```diff
@@ -482,88 +544,118 @@ def multiply(x, y, axis=-1, name=None):
...
+    Returns:
+        N-D Tensor. A location into which the result is stored. Its dimension equals with $x$.
+
+    Examples:
+
+        .. code-block:: python
+
+            import paddle
+            import numpy as np
+
```
Comment (wawltor) [inline python/paddle/tensor/math.py:552]: 同上

File: python/paddle/tensor/math.py [hunk 3]
```diff
@@ -482,88 +544,118 @@ def multiply(x, y, axis=-1, name=None):
...
+        name (str, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
+
+    Returns:
+        N-D Tensor. A location into which the result is stored. Its dimension equals with $x$.
+
+    Examples:
+
+        .. code-block:: python
+
+            import paddle
```
Comment (wawltor) [inline python/paddle/tensor/math.py]: 同上

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (LutaoChu) [reply python/paddle/tensor/math.py:312]: done

File: python/paddle/tensor/math.py (hunk 2 above)
Comment (LutaoChu) [reply python/paddle/tensor/math.py:552]: done

File: python/paddle/tensor/math.py (hunk 3 above)
Comment (LutaoChu) [reply python/paddle/tensor/math.py]: done

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**